internal/cli/atlas/setup/setup_cmd.go

```diff
@@ -563,7 +584,7 @@ func Builder() *cobra.Command {
 		Example: `  # Override default cluster settings like name, provider, or database username by using the command options
   atlas setup --clusterName Test --provider GCP --username dbuserTest`,
 		Hidden: false,
-		Args:   require.NoArgs,
+		Args:   require.MaximumNArgs(1),
```

Comment:
which is the optional arg we're expecting here? (deploymentName is moved to --clusterName <name>)

Reply:
sorry, we don't expect any, removing this (it was during testing)